Tasks/AppCenterDistributeV1/appcenterdistribute.ts

```diff
@@ -455,23 +560,35 @@ async function run() {
         // Prepare symbols
         let symbolsFile = await prepareSymbols(symbolsPaths);
 
-        // Begin release upload
-        let uploadInfo: UploadInfo = await beginReleaseUpload(effectiveApiServer, effectiveApiVersion, appSlug, apiToken, userAgent);
-
-        // Perform the upload
-        await uploadRelease(uploadInfo.upload_url, app, userAgent);
-
-        // Commit the upload
-        let packageUrl = await commitRelease(effectiveApiServer, effectiveApiVersion, appSlug, uploadInfo.upload_id, apiToken, userAgent);
+        let uploadInfo = await beginReleaseUpload(effectiveApiServer, effectiveApiVersion, appSlug, apiToken, userAgent);
```

Comment:
Nit: Could you please add types for all non-constant variables?

Reply:
Fixed in 174b7945b8b9854019132eb489888665c76a434a